cmd/neofs-cli/internal/client/sdk.go

```diff
@@ -19,12 +18,12 @@ var errInvalidEndpoint = errors.New("provided RPC endpoint is incorrect")
 
 // GetSDKClientByFlag returns default neofs-sdk-go client using the specified flag for the address.
 // On error, outputs to stderr of cmd and exits with non-zero code.
-func GetSDKClientByFlag(ctx context.Context, cmd *cobra.Command, endpointFlag string) *client.Client {
+func GetSDKClientByFlag(ctx context.Context, endpointFlag string) (*client.Client, error) {
 	cli, err := getSDKClientByFlag(ctx, endpointFlag)
 	if err != nil {
-		common.ExitOnErr(cmd, "can't create API client: %w", err)
+		return nil, common.WrapError("can't create API client: %w", err)
```

Comment:
i dont think this is a good place to throw `ExitErr`, this is the responsibility of `RunE`s. So, just `return nil, fmt.Errorf(...)`